postgres/datadog_checks/postgres/config.py

```diff
@@ -58,6 +59,12 @@ def __init__(self, instance):
         self.custom_metrics = self._get_custom_metrics(instance.get('custom_metrics', []))
         self.max_relations = int(instance.get('max_relations', 300))
 
+        # Deep Database monitoring adds additional telemetry for statement metrics
+        self.deep_database_monitoring = is_affirmative(instance.get('deep_database_monitoring', False))
+        # Support a custom view when datadog user has insufficient privilege to see queries
+        self.pg_stat_statements_view = instance.get('pg_stat_statements_view', 'pg_stat_statements')
+        self.statement_metric_limits = instance.get('statement_metric_limits', None)
```

Comment:
Doesn't look like this option `statement_metric_limits` was added to the spec / `conf.yaml` ?

At first I was assuming this would store an `int` but it doesn't seem so. It looks like the following format is expected, correct?

```yaml
statement_metric_limits:
    metric1: [<min>, <max>]
    metric2: [<min>, <max>]
    # ...
```

Not sure about what the `(int, int)` tuples mean actually (I've put `min/max` but it's unclear). Docs added in the `conf.yaml` would help.

Reply:
I removed this as a configurable option. I think we can make it configurable as a follow-up when we start thinking about actually applying limits.